client/src/components/Toasts/Toast.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from 'i18n';
+import { Toast as BsToast, ToastBody } from 'reactstrap';
+import classnames from 'classnames';
+import { ToastActions } from './ToastActions';
+import Button from 'components/Button/Button';
+
+/**
+ * Proptype for an individual Toast
+ */
+export const toastPropType = {
+  text: PropTypes.string.isRequired,
+  dismissed: PropTypes.bool.isRequired,
+  type: PropTypes.string.isRequired,
+  actions: PropTypes.arrayOf(PropTypes.shape(Button.propTypes))
+};
+
+/**
+ * Display an individual toast
+ */
+const Toast = ({ type, text, onDismiss, dismissed, actions }) => {
+  // handler for close button
+  const toggle = (e) => {
+    e.preventDefault();
+    onDismiss();
+  };
+
+  const className = classnames('toast', `toast--${type}`, { 'toast--dismissing': dismissed });
+
+  return (
+    <BsToast className={className} isOpen aria-live="assertive" aria-atomic="true">
+      <ToastBody className="toast__body">
+        <Button className="toast__close" icon="cross-mark" noText onClick={toggle}>
```

Comment:
```suggestion
        <Button className="toast__close" icon="cancel" noText onClick={toggle}>
```

Reply:
Sacha has also requested that the close button not have a hover state (e.g. no background) per the designs, but to retain the focus state (outline).

Reply:
Change the icon as requested and remove the background. The X look misaligned a little bit as well, so I tweak the line height of the button so the x would be aligned with the middle of the first line